src/pages/DocumentsPage.js

```diff
@@ -48,7 +48,6 @@ function getDocumentContent(resources, path) {
 
```

Comment:
This file should not be in the PR, as it's basically linting the file and not changing any functionality. Can you unstage this from the PR?

Reply:
Thanks for pointing this out!